README.md

```diff
@@ -252,53 +252,25 @@ config.  If Fedora requires auth, you can also include that in the URL, e.g.:
 
 ## Installing a Development environment
 
-### With Docker
-The development and test stacks use fully contained virtual volumes and bind all services to different ports,
-so they can be running at the same time without issue.
-
-#### External Requirements
-* [Docker](https://store.docker.com/search?offering=community&type=edition) version >= 17.09.0
-
-### Dependency Setup (Mac OSX)
-1. `brew install docker`
-1. `brew install docker-machine`
-1. `brew install docker-compose`
-
-### Starting Docker (Mac OSX)
-1. `docker-machine create default`
-1. `docker-machine start default`
-1. `eval "$(docker-machine env)"`
-
-#### Starting the development mode dependencies
-1. Start Solr, Fedora, and PostgreSQL with `rake docker:dev:daemon` (or `rake docker:dev:up` in a separate
-   shell to run them in the foreground)
-1. Run `rake db:create db:migrate` to initialize the database
-1. Develop!
-1. Run `rake docker:dev:down` to stop the server stack
-   * Development servers maintain data between runs. To clean them out, run `rake docker:dev:clean`
-
-#### To run the test suite with all dependencies in one go
-1. `rake docker:spec`
-
-#### To run the test suite manually
-1. Start Solr, Fedora, and PostgreSQL with `rake docker:test:daemon` (or `rake docker:test:up` in a separate
-   shell to run them in the foreground)
-1. Run `rake db:create db:migrate` to initialize the database
-1. Run the gem's RSpec test suite with `rspec spec` or `rake`
-1. Run `rake docker:test:down` to stop the server stack
-   * The test stack cleans up after itself on exit.
-
-### Without Docker
-
-#### External Requirements
-* PostgreSQL with the uuid-ossp extension.
-  * Note: Enabling uuid-ossp requires database superuser privileges.
-    * From `psql`: `alter user [username] with superuser;`
-
-#### To run the test suite
-1. Start Solr and Fedora servers for testing with `rake server:test`
-1. Run `rake db:create` (First time only)
-1. Run `rake db:migrate`
+For ease of development we use Lando to abstract away some complications of
+using Docker containers for development.
+
+### Running Tests
+
+1. Install the latest released > 3.0 version of Lando from [here](https://github.com/lando/lando/releases).
```

Comment:
Do we want to include Homebrew or other instructions for this? There were some before, and the docker dependencies are still required here. `brew install docker docker-machine docker-compose lando` worked for me.

Reply:
The Lando installer should install docker for you, so I don't think that should be necessary